.github/workflows/sprint-board-maintenance.yml

```diff
@@ -54,3 +54,9 @@ jobs:
           pipenv run moc-sprint-tools -v close-sprint-boards
         env:
           GH_API_TOKEN: ${{ secrets.GH_API_TOKEN }}
+
+      - name: "Create new sprints"
+        run: |
+          pipenv run moc-sprint-tools -v create-sprint-boards
+        env:
+          GH_API_TOKEN: ${{ secrets.GH_API_TOKEN }}
```

Comment:
I wonder if instead of running every 30 minutes with the regular maintenance tasks if there should be a separate workflow for this task?

Reply:
To be honest, I'm starting to question the whole need for automation to create sprint boards. Last time we did planning we preferred to not have the cards copied over, and that is the main thing that this brings. Otherwise *just* creating a board and 4 columns takes 10 seconds (which need to be spent anyway setting up the board automation since GitHub API doesn't support that).